src/assets/stylesheets/index.css

```diff
@@ -3,6 +3,7 @@
 @tailwind utilities;
 
 body {
+  background-color: #b2dfff;
```

Comment:
why are you adding this color as the default background for all pages, some pages will remain white as per the design given?

Reply:
I will create another div with this bg color instead of giving it directly to the body before merging